influxdb_iox/src/commands/remote.rs

```diff
@@ -0,0 +1,43 @@
+//! This module implements the `remote` CLI command
```

Comment:
```suggestion
//! This module implements the `remote` CLI command (NG)
```

It is getting somewhat confusing what CLI commands are for NG and what are for OG. The sooner we can remove the OG code the better I think